qabelbox/src/main/java/de/qabel/qabelbox/index/AndroidIndexSyncService.kt

```diff
@@ -58,6 +60,7 @@ class AndroidIndexSyncService() : IntentService(AndroidIndexSyncService::class.j
             }
         } catch (ex: Throwable) {
             error("Error syncing with index. Action: ${intent.action}", ex)
+            crashSubmitter.submit(ex)
```

Comment:
where does this come from?
and where is the difference between the CrashReporter and the CrashSubmitter (especially as you seem to need both)?